src/app.module.ts

```diff
@@ -14,7 +17,13 @@ import { PrismaService } from './prisma.service';
     HttpModule,
     CacheModule.register(),
   ],
-  controllers: [CmcController],
-  providers: [PrismaService, CmcService, ConfigService],
+  controllers: [CmcController, SendgridController],
```

Comment:
**nit:** `SendGridConroller`, should be a part of `SendGridModule`. `CmcController` should be a part of `CmcModule`. Then these modules should be imported into `AppModule`